redwood-treehouse-host-composeui/src/commonMain/kotlin/app/cash/redwood/treehouse/composeui/TreehouseContent.kt

```diff
@@ -92,3 +96,6 @@ public fun <A : AppService> TreehouseContent(
     treehouseView.children.render()
   }
 }
+
+@Composable
+internal expect fun platformOnBackPressedDispatcher(): OnBackPressedDispatcher
```

Comment:
This function and its `actual` definitions are identical to that of `RedwoodContent`. When `TreehouseContent` is a wrapper around `RedwoodContent`, this function and its `actual`'s can be removed.